compiler/passes/normalize.cpp

```diff
@@ -2027,6 +2030,14 @@ static void normalizeTypeAlias(DefExpr* defExpr) {
   Expr*   type = defExpr->exprType;
   Expr*   init = defExpr->init;
 
+  if (type == NULL && init == NULL) {
+    // Don't worry about compiler temporaries
+    INT_ASSERT(var->hasFlag(FLAG_TEMP));
+    return;
+  }
+
+  // TODO: call findInitPoints
```

Comment:
Is it still a TODO?
If so, how can it possibly work without calling it?
Please double-check or update the comment.

Reply:
It's a TODO for future work for getting split init working for type aliases ( #14565)